config/config.go

```diff
@@ -52,11 +55,11 @@ type BlockManagerConfig struct {
 	// MaxProofTime defines the max time to be idle, if txs that requires proof were included in last block
 	MaxProofTime time.Duration `mapstructure:"max_proof_time"`
 	// BatchSubmitMaxTime is how long should block manager wait for before submitting batch
-	BatchSubmitMaxTime time.Duration `mapstructure:"batch_submit_max_time"`
-	// MaxBatchSkew is the number of batches which are waiting to be submitted. Block production will be paused if this limit is reached.
-	MaxBatchSkew uint64 `mapstructure:"max_supported_batch_skew"`
+	BatchSubmitTime time.Duration `mapstructure:"batch_submit_max_time"`
+	// BatchSkewBlocks is the number of blocks which are waiting to be submitted. Block production will be paused if this limit is reached.
+	BatchSkewBlocks uint64 `mapstructure:"batch_skew_blocks"`
```

Comment:
In general not sure the idea of measuring skew by blocks is good (see my other comment). regradless I'd change the name to
```suggestion
	MaxBlockSkew uint64 `mapstructure:"max_block_skew"`
```

Reply:
rolled back to batch skew